pkgs/by-name/op/openmpi/package.nix

```diff
@@ -26,6 +26,7 @@
   cudaPackages,
   # Enable the Sun Grid Engine bindings
   enableSGE ? false,
+  enablePRRTE ? true,
```

Comment:
What is the motivation to make it configurable? You can of course disable prrte here explicitly, but then it will still be built with the openmpi internally packaged version for prrte (i.e. prrte is never really disabled, only the external dependency). Also: the changes in `mlx` do not make use of this the "disable" feature as far as I can tell.

Reply:
Is there any downside to making it configurable?

Reply:
Also, to clarify: `mlx` does depend on `prrte` being enabled in order to make of use `openmpi`

Without this configure flag enabled, `openmpi` doesn't build and install the `mpirun` executable, which causes the downstream `mlx` build because the MPI configuration step of the `mlx` build looks for the `mpirun` executable.

Reply:
This discussion is too messy.

> Is there any downside to making it configurable?

Please read carefully what @markuskowa wrote:

> prrte is never really disabled, only the external dependency).

Reply:
I read it, but I don't believe what @markuskowa said is relevant.  You can try setting `enablePRRTE = False` and see for yourself that the OpenMPI support of `mlx` fails, so there is clearly a difference between enabling and disabling it.  If you think I misunderstood something then please elaborate or rephrase instead of restating what was already said (which is not helpful)

Reply:
@Gabriella439 Ok, I correct myself: the switch turns off prrte and you are left with a pmix-only version of mpi. 
However, my point here is twofold: The introduction of the "enablePRRTE" feature is unrelated to the topic of this PR (i.e. the improvement of mlx). This is why I asked in the first place.
From your answer, I take it that there is no clear motivation for introducing that switch. If someone comes with a clear argument why a build without PRRTE would be a good idea, I am okay with putting it in.
IMHO we should not add code that has no clear purpose or motivation, as every line of code needs to be maintained in the future (and openmpi is already a big beast).